modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -787,25 +787,37 @@ See xref:configuration/files/encryption_configuration.adoc[encryption configurat
 
 `encryption:recreate-master-key` decrypts the ownCloud file system, replaces the existing master key with a new one, and encrypts the entire ownCloud file system with the new master key. Given the size of your ownCloud filesystem, this may take some time to complete. However, if your filesystem is quite small, then it will complete quite quickly. The `-y` switch can be supplied to automate acceptance of user input.
 
-[[federation-sync]]
-== Federation Sync
+== Federated Shares
 
-Synchronize the address books of all federated ownCloud servers.
+=== Synchronize All Federated Server Address Books
+
+Servers connected with federation shares can share user address books and auto-complete usernames in share dialogs. 
+Use this command to synchronize federated servers:
 
 [source,console]
 ----
-federation:sync-addressbooks  Synchronizes address books of all federated clouds
+sudo -u www-data php occ federation:sync-addressbooks
 ----
 
-Servers connected with federation shares can share user address books,
-and auto-complete usernames in share dialogs. Use this command to
-synchronize federated servers:
+NOTE: This command is only available when the "Federation" app (`federation`) is enabled.
 
-....
-sudo -u www-data php occ federation:sync-addressbooks
-....
+=== Poll Incoming Federated Shares For Updates
 
-NOTE: This command is only available when the "Federation" app (`federation`) is enabled.
+This command must be used if federated shares are being updated with a Desktop client or on a regular basis.
```

Comment:
@phil-davis could you please proof-read this sentence? I'm not sure whether it is clear enough.

Reply:
Maybe:
This command must be used if federated shares are being referenced by Desktop clients but not regularly accessed via the webUI.

Reply:
@phil-davis, that works perfectly for me. Thanks.

Reply:
@phil-davis, @VicDeo the sentence has been updated.